content/developer/troubleshooting/sso.md

```diff
@@ -0,0 +1,27 @@
+---
+title: SSO Debugging
+tags:
+- Features
+- Troubleshooting
+- Developers
+category: developer
+menu:
+  developer:
+    identifier: sso
+    parent: troubleshooting
+aliases:
+- /developers/troubleshooting/sso
```

Comment:
Whats the purpose of this alias? Did we have a previous SSO troubleshooting page?